tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -1457,13 +1457,26 @@ def activate_filesystems_test(self, patched_conf):
         storage.assert_not_called()
 
     @patch("pyanaconda.core.util.mkdirChain")
-    @patch("pyanaconda.core.util.execWithRedirect")
-    def mount_filesystems_test(self, execute, mkdir):
+    @patch("pyanaconda.core.util._run_program")
+    @patch("os.makedirs")
+    @patch("blivet.util._run_program")
+    def mount_filesystems_test(self, blivet_run_program, makedirs, core_run_program, mkdirChain):
         """Test MountFilesystemsTask."""
         storage = create_storage()
         storage._bootloader = Mock()
-        execute.return_value = 0
+        blivet_run_program.return_value = (0, "")
+        core_run_program.return_value = (0, "")
         MountFilesystemsTask(storage).run()
+        # created the mount points
+        makedirs.assert_any_call('/mnt/sysimage/dev', 0o755)
```

Comment:
Not important, but if I wanted to be pedantic, I'd ask if you could go with `assert_called_once_with` instead.

Reply:
But that's not true -- It gets called with a lot of other mount points, like /mnt/sysimage/proc and /sys, and /selinux etc. I don't think it makes much sense to replicate that entire list here, and e.g. the /selinux submount is conditional (it doesn't happen in a container with SELinux disabled). So I picked /dev as a representative example, and leave the rest to blivet -- we are not testing the blivet internals here.